### PR TITLE
Update Dockerfile.scancode

### PR DESCRIPTION
**Pull request by KerinPithawala**
Solves issue #1013

Kindly review my PR.

### CODE REVIEW

File: docker/Dockerfile.scancode [hunk 1]
```diff
@@ -13,14 +13,15 @@ RUN apt-get update \
     build-essential \
     && rm -rf /var/lib/apt/lists/*
 
-RUN pip install --no-warn-script-location --prefix=/install \
-    tern
+
+
```
Comment (rnjudge) [inline docker/Dockerfile.scancode:17]: Can you remove these blank lines?

File: docker/Dockerfile.scancode [hunk 2]
```diff
@@ -13,14 +13,15 @@ RUN apt-get update \
...
-    tern
+
+
 
 RUN pip install --no-warn-script-location --prefix=/install \
     tern \
     scancode-toolkit[full]
 
 
+
```
Comment (rnjudge) [inline docker/Dockerfile.scancode:24]: Remove this one too

**Comment by rnjudge** [comment]
Hi @KerinPithawala -- looks like this issue was already resolved with https://github.com/tern-tools/tern/pull/1015 but the issue was left off of the commit message so the issue didn't get closed. Sorry about that! We have lots more issues, however, so if there's another one that looks fun to you feel free to grab it!

File: docker/Dockerfile.scancode (hunk 1 above)
Comment (rnjudge) [inline docker/Dockerfile.scancode:17]: You can remove these extra blank lines :)

File: docker/Dockerfile.scancode (hunk 2 above)
Comment (rnjudge) [inline docker/Dockerfile.scancode:24]: same with this one

**Comment by rnjudge** [comment]
> Hi @KerinPithawala -- looks like this issue was already resolved with #1015 but the issue was left off of the commit message so the issue didn't get closed. Sorry about that! We have lots more issues, however, so if there's another one that looks fun to you feel free to grab it!

Oh wait, I see what your PR is doing -- Thank you! This PR is fixing a duplicate line that's currently there. Sounds good! Before merging this I will need you to update your commit message in accordance with our [commit message guidelines](https://github.com/tern-tools/tern/blob/main/CONTRIBUTING.md#commit-message-format) in CONTRIBUTING.md, something like:

```
Fix duplicate line in Dockerfile.scancode

This commit removes a duplicate line of code in the Dockerfile.scancode
file that was introduced by a previous commit.

Resolves #1013 

Signed-off-by: Kerin Pithawala <your email>
```

Also, could you remove the extra blank lines you added in your commit? You can do this by making your changes to the file, run `git add docs/Dockerfile.scancode`, `git commit --amend` and then update your commit message and run `git push -f origin main`

**Comment by KerinPithawala** [comment]
Yeah, sure. I will make the necessary changes.

**Comment by KerinPithawala** [comment]
Kindly review the changes. Thank you for your guidelines.

**Pull request merged**